packages/conf-blas/conf-blas.1/opam

```diff
@@ -6,7 +6,8 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 dev-repo: "https://github.com/ocaml/opam-repository.git"
 license: "GPL-2.0"
 build: [
-  ["sh" "-exc" "cc $CFLAGS test.c -lblas"] {os != "darwin" & os != "win32"}
+  ["sh" "-exc" "cc $CFLAGS test.c -lblas"] {os != "darwin" & os != "freebsd" & os != "win32"}
+  ["sh" "-exc" "${CC:-gcc} $CFLAGS test.c -lblas $LDFLAGS"] {os = "freebsd"}
```

Comment:
Hm, isn't that a `bash`-ism used there?  This logic might be better put into a shell script inside the package, as the same problem probably applies to several BSD variants (with /usr/local for -lblas)?